lib/index.js

```diff
@@ -351,6 +368,10 @@ class Study extends EventTarget {
       case "downgrade":
         emit(this, "change", "normal-shutdown")
         break;
+
+      // custom, study-specific reason for ending study
+      default:
```

Comment:
I wanted to create a way to use multiple custom uninstall events, so that the `reason` parameter in the `shutdown` method can be set to the name of a given reason for ending the study. I think that your suggestion would prevent this, as we would have to have `reason == uninstall-custom-reason` for the switch statement to work properly. 

What do you think would be the best way to handle multiple custom uninstall event types? Should we add an option argument in the `shutdown` method for specifying the custom uninstall event name?